web_src/svg/octicon-custom-pin-off.svg

```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" viewBox="0 0 16 16" width="16" height="16"><path d="M14.779 13.721 11.061 10l2.205-2.204c.967-.968.5-2.623-.832-2.94l-.73-.174 2.645-2.625a.7.7 0 0 0-.986-.994L1.023 13.308a.7.7 0 0 0 .986.994l2.662-2.642.184.773c.317 1.332 1.972 1.799 2.94.832L10 11.061l3.719 3.719a.751.751 0 0 0 1.08.001.747.747 0 0 0-.02-1.06zm-8.045-1.517a.25.25 0 0 1-.42-.121l-.396-1.661 4.532-4.498 1.634.389a.25.25 0 0 1 .12.42l-5.47 5.471zM4.534 7.527a.86.86 0 0 0-.078-.082 3.634 3.634 0 0 0-1.262-.912L1.866 5.92a.252.252 0 0 1-.073-.404l3.722-3.722a.251.251 0 0 1 .405.072l.612 1.327c.27.585.409.881.875 1.293l.012.012c.007.005.012.012.019.018l.003-.003a.74.74 0 0 0 .496.197c.41 0 .742-.332.742-.742a.73.73 0 0 0-.147-.421l.007-.007c-.211-.294-.442-.538-.644-.975l-.613-1.327A1.75 1.75 0 0 0 4.456.734L.734 4.456a1.75 1.75 0 0 0 .504 2.826l1.327.613c.428.197.59.272.882.626a.742.742 0 0 0 1.269-.522.724.724 0 0 0-.139-.409"/></svg>
```

Comment:
You can use `octicon-pin-slash`:

https://github.com/go-gitea/gitea/blob/main/public/assets/img/svg/octicon-pin-slash.svg